javascript/lang/security/audit/sqli/node-mysql-sqli.yaml

```diff
@@ -29,7 +31,14 @@ rules:
     pattern-sources:
       - patterns:
           - pattern-inside: function ... (..., $Y,...) {...}
-          - focus-metavariable: $Y
+          - pattern: $Y
+          - pattern-not-inside: |
+              function ... (..., $Y: number,...) {...}
+          - pattern-not-inside: $Y.query
```

Comment:
It would be more efficient to keep the `focus-metavariable: $Y` and instead use sanitizers, or else enumerate the cases `$Y.FOO` that are problematic.